Modules/Filtering/ImageStatistics/include/itkLabelOverlapMeasuresImageFilter.hxx

```diff
@@ -375,17 +375,18 @@ LabelOverlapMeasuresImageFilter<TLabelImage>::GetFalsePositiveError() const -> R
 {
   RealType numerator = 0.0;
   RealType denominator = 0.0;
+
+  LabelImagePointer sourceImg = const_cast<TLabelImage *>(this->GetSourceImage());
```

Comment:
Oops, I now see this second commit. So disregard the comment about moving the lines.